i18n/en.json

```diff
@@ -1,16 +1,16 @@
 {
-    "commtech-commons-speedy-header": "{{PLURAL:$1|A Commons file|Commons files}} used on this page {{PLURAL:$1|has|have}} been nominated for speedy deletion",
-    "commtech-commons-speedy-body-start": "The following Wikimedia Commons {{PLURAL:$1|file|files}} used on this page {{PLURAL:$1|has|have}} been nominated for speedy deletion:",
+    "commtech-commons-speedy-header": "{{PLURAL:$1|A Commons file|Commons files}} used on this page or {{PLURAL:$1|its|their}} Wikidata {{PLURAL:$1|item|items}} {{PLURAL:$1|has|have}} been nominated for speedy deletion",
```

Comment:
The pluralization is used if there are multiple files used on one page, and the one page can have only one Wikidata item. So I don't think we need the second bit of pluralization. This should do it:
```
{{PLURAL:$1|A Commons file|Commons files}} used on this page or its Wikidata item {{PLURAL:$1|has|have}} been nominated for speedy deletion
```

Same for other messages below

Reply:
Cool. Fixing now.